docs/tutorial/api.ipynb

```diff
@@ -0,0 +1,2751 @@
+       "      <td>3grt</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>3grt__1__1.A_2.A__2.C</td>\n",
+       "      <td>3grt</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>1grx__1__1.A__1.B</td>\n",
+       "      <td>1grx</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1357899</th>\n",
+       "      <td>4lpn__1__10.A_24.A_3.A__24.X</td>\n",
+       "      <td>4lpn</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1357900</th>\n",
+       "      <td>2lp3__1__1.A__1.C</td>\n",
+       "      <td>2lp3</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1357901</th>\n",
+       "      <td>2lp3__1__1.A__1.D</td>\n",
+       "      <td>2lp3</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1357902</th>\n",
+       "      <td>2lp3__1__1.B__1.E</td>\n",
+       "      <td>2lp3</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1357903</th>\n",
+       "      <td>2lp3__1__1.B__1.F</td>\n",
+       "      <td>2lp3</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>1357904 rows × 2 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                            system_id entry_pdb_id\n",
+       "0               3grt__1__1.A_2.A__1.B         3grt\n",
+       "1               3grt__1__1.A_2.A__1.C         3grt\n",
+       "2               3grt__1__1.A_2.A__2.B         3grt\n",
+       "3               3grt__1__1.A_2.A__2.C         3grt\n",
+       "4                   1grx__1__1.A__1.B         1grx\n",
+       "...                               ...          ...\n",
+       "1357899  4lpn__1__10.A_24.A_3.A__24.X         4lpn\n",
+       "1357900             2lp3__1__1.A__1.C         2lp3\n",
+       "1357901             2lp3__1__1.A__1.D         2lp3\n",
+       "1357902             2lp3__1__1.B__1.E         2lp3\n",
+       "1357903             2lp3__1__1.B__1.F         2lp3\n",
+       "\n",
+       "[1357904 rows x 2 columns]"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from plinder.core import get_manifest\n",
+    "get_manifest()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Query protein similarity\n",
+    "The are three kinds of similarity datasets we provide:\n",
+    "- Similarity between ligand bound structures (`holo`)\n",
+    "- Similarity between ligand bound and unbound protein structures (`apo`)\n",
+    "- Similarity between ligand bound and Alphafold predicted structures (`pred`)\n",
+    "Any of these could be specified with {func}`query_protein_similarity()`"
```

Comment:
Need to add some kind of warning here, running `query_protein_similarity(search_db="holo")` with only similarity > 50 can easily take >500G RAM

Reply:
For the tutorial, the config is set so that it queries a small set. However, I agree that for the full set, users need to know what's coming.